themes/ohcoe/static/js/scripts.js

```diff
@@ -68,4 +69,30 @@ document.addEventListener('DOMContentLoaded', function() {
     if (scoreContainer && domainScoreGrowth) {
         scoreContainer.innerHTML = 'You have this much growth: <span id="score-container">' + domainScoreGrowth + '%</span>';
     }
+
+    /* Demographic Questions */
+    document.querySelectorAll('.demographic-questions input[type="text"]').forEach(function(el){
+        el.addEventListener('focus', function(e){
```

Comment:
Some linting things here. Is `eslint` turned on?

Reply:
Nope not yet.  Given the extent of the js here, I can add that as a commit to this PR.

Reply:
I thought I'd be clever and not use jQuery for this site but: